Modules/Loadable/Segmentations/Logic/vtkSlicerSegmentationGeometryLogic.h

```diff
@@ -57,6 +57,10 @@ class VTK_SLICER_SEGMENTATIONS_LOGIC_EXPORT vtkSlicerSegmentationGeometryLogic :
   /// Use this calculated permutation for updating spacing widget from geometry and interpreting spacing input
   void ComputeSourceAxisIndexForInputAxis();
 
+  /// Sets the computed reference image geometry into the input segmentation node
+  /// (in the ReferenceImageGeometry conversion parameter and node reference).
+  bool SetReferenceImageGeometryInSegmentationNode();
+
   /// Resample labelmap representation the input segmentation node according to the current
   /// output geometry setting.
```

Comment:
```suggestion
  /// output geometry setting.
  ///
  /// \sa SetReferenceImageGeometryInSegmentationNode()
```

Reply:
It may also be worth clarifying the docstring of `SetReferenceImageGeometryInSegmentationNode` to explain how it relates to "the current output geometry setting"